addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -226,6 +226,7 @@ body.editor_enable.editor_has_snippets {
 #oe_snippets {
     @include o-w-preserve-btn;
 
+    @include o-input-number-no-arrows();
```

Comment:
Do we still need the 2 other @include of this mixin with this change ?

Reply:
Unless I'm mistaken there are only two occurrences of this: once here so that the editor's numeric inputs don't have the two ugly arrows to do ++ or --. The only other occurrence I see is for the donation snippet which offers a number input on the **client side**. If I remove it, the donation snippet will have the ugly arrows.
![arrows](https://user-images.githubusercontent.com/78849981/156006344-4426614c-41cf-4387-9a59-1bb7615341e7.png)


Tell me if I'm missing something :)

Reply:
Indeed we still need it for the donation nsippet, but maybe we no longer need the other in the options panel for donation snippet.. I let you check ;)

Reply:
Was there a conclusion here?

Reply:
If I am not it's better to keep the both call to the mixin. The first one to disable the arrows for the block donation and the second one to disable the arrows in the editor. I think it's clearer this way. We could call this mixin only once with a selector that takes both cases into account but it would seem less clear IMHO...

Reply:
ok, I was talking about removing the "include" from the "we-list table" css rule but I see you did. So it's ok :+1: